### PR TITLE
modify abwarten, add weiterwarten and beten

### PR DESCRIPTION
**Pull request by jsgros**
Please review this, @casey-collab

### CODE REVIEW

File: german/abwarten/abwarten.md [hunk 1]
```diff
@@ -3,4 +3,5 @@ In der Tat lösen sich so einige Probleme von selbst, wenn man einfach nur lange
 Bei genauerer Betrachtung ist die aktuelle allerdings keine solche Situation, in der die sich 
 ergebende Lösung besonders angenehm oder reizvoll erscheint. 
 
-[Daher bietet es sich an doch noch eine andere Entscheidung zu treffen...](../ruderboot.md)
\ No newline at end of file
+[Daher bietet es sich an doch noch eine andere Entscheidung zu treffen...](../ruderboot.md)
+[Oder aber sie warten trotzdem weiter ab...](../weiterwarten.md)
```
Comment (casey-collab) [inline german/abwarten/abwarten.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

Comment (casey-collab) [inline german/abwarten/abwarten.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: german/weiterwarten/weiterwarten.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Eventuell war das nicht die cleverste Option, mittlerweile steht ihnen das Wasser bis zum Hals.
+
+[Jetzt nehmen sie das - ähem - Ruder in die Hand...](../ruderboot.md)
+[Sie beginnen zu beten...](../beten.md)
```
Comment (casey-collab) [inline german/weiterwarten/weiterwarten.md]: This link is also invalid.

Comment (casey-collab) [inline german/weiterwarten/weiterwarten.md]: This link is also on the line directly after another link.

File: german/beten/beten.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Kein Anschluss unter dieser Nummer.
```
Comment (casey-collab) [inline german/beten/beten.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: german/weiterwarten/weiterwarten.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+Eventuell war das nicht die cleverste Option, mittlerweile steht ihnen das Wasser bis zum Hals.
```
Comment (casey-collab) [inline german/weiterwarten/weiterwarten.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by jsgros** [comment]
Please review "correct broken links", @casey-collab

File: german/abwarten/abwarten.md [hunk 5]
```diff
@@ -3,4 +3,5 @@ In der Tat lösen sich so einige Probleme von selbst, wenn man einfach nur lange
 Bei genauerer Betrachtung ist die aktuelle allerdings keine solche Situation, in der die sich 
 ergebende Lösung besonders angenehm oder reizvoll erscheint. 
 
-[Daher bietet es sich an doch noch eine andere Entscheidung zu treffen...](../ruderboot.md)
\ No newline at end of file
+[Daher bietet es sich an doch noch eine andere Entscheidung zu treffen...](../ruderboot.md)
+[Oder aber sie warten trotzdem weiter ab...](weiterwarten/weiterwarten.md)
```
Comment (casey-collab) [inline german/abwarten/abwarten.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

Comment (casey-collab) [inline german/abwarten/abwarten.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: german/weiterwarten/weiterwarten.md [hunk 6]
```diff
@@ -0,0 +1,4 @@
...
+
+[Jetzt nehmen sie das - ähem - Ruder in die Hand...](../ruderboot.md)
+[Sie beginnen zu beten...](beten/beten.md)
```
Comment (casey-collab) [inline german/weiterwarten/weiterwarten.md]: This link is also invalid.

Comment (casey-collab) [inline german/weiterwarten/weiterwarten.md]: This link is also on the line directly after another link.

File: german/beten/beten.md (hunk 3 above)
Comment (casey-collab) [inline german/beten/beten.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: german/weiterwarten/weiterwarten.md (hunk 4 above)
Comment (casey-collab) [inline german/weiterwarten/weiterwarten.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**